### PR TITLE
Add support for retrieve info for Label

### PR DESCRIPTION
**Pull request by jerrysf**
Initial version for supporting get tied jobs under specific label

### CODE REVIEW

**Comment by jerrysf** [comment]
Looks like the test failure is not related to my change?

File: jenkinsapi/label.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    def __str__(self):
+        return '%s' % (self.labelname)
+
+    def get_jenkins_obj(self):
+        return self.jenkins
+
+    def is_online(self):
+        return not self.poll(tree='offline')['offline']
+
+    def get_tied_jobs(self):
```
Comment (lechat) [inline jenkinsapi/label.py]: Either this shall return list of ``Job`` objects, or it shall be named differently. Just to make API a bit helpful: if something returns jobs, it shall return a list of objects and not just names.

I'd have it as ``tied_jobs()`` return a list of Jobs and ``tied_job_names`` returning job names.

Comment (jerrysf) [inline jenkinsapi/label.py]: Thanks, just added tied_job_names() and updated tied_jobs()

File: jenkinsapi/label.py [hunk 2]
```diff
@@ -37,6 +37,12 @@ def is_online(self):
 
     def get_tied_jobs(self):
         """
+        Get a list of jobs.
+        """
+        return self.poll(tree='tiedJobs[name,url,color]')['tiedJobs']
```
Comment (lechat) [inline jenkinsapi/label.py]: This does not return a list of ``Job`` objects...

Comment (jerrysf) [inline jenkinsapi/label.py]: Sorry, updated again...

File: jenkinsapi/jenkins.py [hunk 3]
```diff
@@ -222,6 +223,10 @@ def delete_view_by_url(self, str_url):
         self.poll()
         return self
 
+    def get_label_by_url(self, str_label_url):
```
Comment (salimfadhley) [inline jenkinsapi/jenkins.py]: Python projects generally do not use these Microsoft-style type prefixes on variable names. If we really want to provide guidance to users we might start using type-hints in the next major version. This variable should be called label_url.

**Comment by salimfadhley** [comment]
Do you think we could have a system-test for this as well? It seems like the sort of thing that might easily change in the Jenkins REST API. I think all we'd need to do to test this one is to generate 2 different jobs with two different labels. Verify that we can retrieve each of the jobs by their labeling alone.

File: jenkinsapi/jenkins.py (hunk 3 above)
Comment (jerrysf) [inline jenkinsapi/jenkins.py]: Thanks for comment, have updated to label_url

File: jenkinsapi/jenkins.py [hunk 4]
```diff
@@ -222,6 +223,10 @@ def delete_view_by_url(self, str_url):
         self.poll()
         return self
 
+    def get_label_by_url(self, label_url):
```
Comment (lechat) [inline jenkinsapi/jenkins.py]: There is no API to get all labels, AFAIK, but all labels are accessible via `http://jenkins:port/label/label_name`. Why not have `get_label` function, that will return `Label` object by label name?

Comment (jerrysf) [inline jenkinsapi/jenkins.py]: Yeah, I noticed that I could just use the baseurl, have updated get_label_by_url to get_label function. :-)

**Pull request merged**